src/api/app/views/webui2/webui/package/_tabs.html.haml

```diff
@@ -0,0 +1,15 @@
+.bg-light
+  %ul.nav.nav-tabs.pt-2.px-3
```

Comment:
This looks funky on mobile...

![screen shot 2018-07-25 at 16 46 39](https://user-images.githubusercontent.com/514785/43208312-6eead5b8-902a-11e8-8093-d427ad5ac28f.png)

Reply:
Indeed, what should we do then? I'm not sure...

Reply:
With borders, it would be already better... I'll think about other possible solutions too.

Reply:
welcome to the wonderful world of web development 🤡

there are many patterns solving this problem. the last time I checked the most popular where

1. turning tabs into a vertical accordion 
2. move the tabs that cause the 'overflow' to a dropdown